setup.py

```diff
@@ -26,7 +26,7 @@
 import os
 import re
 import setuptools
-from distutils import log
+from distutils import log  # pylint: disable=wrong-import-order
```

Comment:
I assume there is a reason we don't put it into the correct order.

Reply:
Yes. setuptools needs to be import before distutils. If imported afterwards, it issues a UserWarning that it might not work. I'll add a comment to that extent.

Reply:
DONE